### PR TITLE
[HFEYP-621] Hide support articles on the landing page if feature not enabled

### PR DESCRIPTION
**Pull request by ebrett**
## Ticket and context

[HFEYP-621](https://dfedigital.atlassian.net/browse/HFEYP-621)

You can also now set an ENV var to enable a feature

`cf set-env eyfs-cms-preprod SUPPORT_ARTICLES enabled`

### Code quality checks
- [ ] All commit messages are meaningful and true

### CODE REVIEW

File: script/network-policies [hunk 1]
```diff
@@ -0,0 +1,6 @@
+cf target -s eyfs-dev
+cf add-network-policy eyfs-cms-dev eyfs-clamav-rest-private -s eyfs-prod --protocol tcp --port 9000
+#cf add-network-policy eyfs-cms-review-pr-478 eyfs-clamav-rest-private -s eyfs-prod --protocol tcp --port 9000
+cf target -s eyfs-pre-prod
+cf add-network-policy eyfs-cms-preprod eyfs-clamav-rest-private -s eyfs-prod --protocol tcp --port 9000
+
```
Comment (reggieb) [inline script/network-policies:6]: I'm not sure what this file has to do with this change - has it slipped in from something else?

File: app/views/content/index.html.erb [hunk 2]
```diff
@@ -32,19 +32,21 @@
...
-      <h2 id="get-support-and-guidance" class="govuk-heading-l eyfs-top-nav-anchors">Get support and guidance</h2>
-      <ul class="govuk-grid-row eyfs-card-group">
-          <li class="govuk-grid-column-one-half eyfs-card-group__item">
-            <div class="eyfs-card eyfs-card--clickable eyfs-card--link">
-              <div class="eyfs-card__content">
-                <a class="govuk-link govuk-!-font-size-19 govuk-!-font-weight-bold govuk-link--no-visited-state" href="/articles">Support articles</a>
-                <svg class="eyfs-card--icon" width="30" height="30" viewBox="0 0 30 30" fill="none" xmlns="http://www.w3.org/2000/svg">
-                  <path d="M9 4.49992L20.0217 15.5213L9 26.5427" stroke="#1D70B8" stroke-width="2"/>
-                </svg>
+      <%- if Feature.enabled?(:support_articles) %>
```
Comment (reggieb) [inline app/views/content/index.html.erb:35]: Extremely minor: `<%-` is the same as `<%` in the current version of erb, so the `-` is redundant.

**Review by reggieb — COMMENTED** [review]
Nice. It just looks to me like extra file has slipped in.

File: script/network-policies (hunk 1 above)
Comment (ebrett) [reply script/network-policies:6]: yes, this is a helper script I added - will remove

**Review by reggieb — APPROVED** [review]
Good stuff

**Pull request merged**